Building Sniffer

I'm trying to build Sniffer based on the instructions provided in the README. After running the commands `git submodule init && git submodule update`, I executed this command `./gradlew packages`.  I received the following error 

``` 
$ ./gradlew packages 
Starting a Gradle Daemon (subsequent builds will be faster)\
FAILURE: Build failed with an exception.
```

After running the command with the switch `./gradlew --stacktrace packages`, I received the entire error trace, but I think this is the main cause 
`
Caused by: java.io.IOException: Cannot run program "mvn" (in directory "/Users/think/Downloads/codeSmell/Sniffer/GitHubMiner"): error=2, No such file or directory
`
Where I already have maven installed 
```
$ mvn -version
Apache Maven 3.6.1 (d66c9c0b3152b2e69ee9bac180bb8fcc8e6af555; 2019-04-04T14:00:29-05:00)
Maven home: /Applications/apache-maven-3.6.1
Java version: 1.8.0_181, vendor: Oracle Corporation, runtime: /Library/Java/JavaVirtualMachines/jdk1.8.0_181.jdk/Contents/Home/jre
Default locale: en_US, platform encoding: UTF-8
OS name: "mac os x", version: "10.14.5", arch: "x86_64", family: "mac"
```

And I have that directory `/Users/think/Downloads/codeSmell/Sniffer/GitHubMiner`

--- a/build.gradle
+++ b/build.gradle
@@ -10,28 +10,8 @@ buildscript {
     }
 }
 
-
-task gitHubMinerMavenPackage(type: Exec) {
-    workingDir './GitHubMiner'
-
-    executable "mvn"
-    args 'package'
-}
-
-task gitHubMinerClean(type: Exec) {
-    workingDir './GitHubMiner'
-
-    executable "mvn"
-    args 'clean'
-}
-
 task packages {
     subprojects.each { dependsOn("${it.name}:shadowJar") }
-    packages.dependsOn(gitHubMinerMavenPackage)
-}
-
-task clean {
-    dependsOn(gitHubMinerClean)
 }
 
 subprojects {

